### PR TITLE
Added Things, fixed Things

### PR DESCRIPTION
**Pull request by jani270**
- Added Wiki Links 
- Fixed Rare Lion (hopefully)
- Fixed Undead Bow (tf)
- Added new Enchants to the Maxed Book 
- Fixed some Slayer Items (eg. Mastiff)
- Removed Recipe from Maddox Bat Phone (https://hypixel.net/threads/skyblock-known-issues-and-information.2135677/)
- Added Text Req. for Slayer Items
- Added Recipe for Pack Armor 

_pls gib Reward for creative Title and Mod Rank on Discord_

### CODE REVIEW

**Comment by IRONM00N** [comment]
I am in the middle of making a commit, I will accept after I push my commit.

**Comment by jani270** [comment]
k

File: items/MAXED_ENCHANT_BOOK.json [hunk 1]
```diff
@@ -1,47 +1,67 @@
...
   "damage": 0,
   "lore": [
     "§aThis is to test custom enchant color.",
```
Comment (IRONM00N) [inline items/MAXED_ENCHANT_BOOK.json:7]: Have you tested if lore this long displays ingame?

Comment (jani270) [reply items/MAXED_ENCHANT_BOOK.json:7]: http://prntscr.com/wmp1km

Comment (jani270) [reply items/MAXED_ENCHANT_BOOK.json:7]: wait i send you the file on dc so you can test it

**Pull request merged**